pages/tickets/[...slug].js

```diff
@@ -0,0 +1,53 @@
+import { useRouter } from "next/router";
+import { Fragment } from "react";
+import Link from "next/link";
+import { getSpecifikScreenings, getMovieList } from "../../helpers/getScreenings";
+import ScreeningsList from "../../components/screenings/ScreeningsList";
+import ScreeningSearch from "../../components/screenings/Screening-search";
+
+function FilteredTicketsPage(props) {
+    const router = useRouter();
+    function onSearchHandler(name) {
+        const fullPath = `/tickets/${name}`;
+        router.push(fullPath)
+    }
+
+    if (props.screenings.length > 0) {
+        return <Fragment>
```

Comment:
This has the same formatting issue that I've commented on elsewhere.

Also a hint: You can use `<>` and `</>` as shorthands to open/close fragments.

This is how I would have written this code:

```tsx
return (
  <>
    <h1>Biljetter</h1>
    <ScreeningSearch onSearch={onSearchHandler} list={props.list} filter={props.name} />
    <ScreeningsList screenings={props.screenings} />
  </>
);
```